device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py

```diff
@@ -83,7 +84,8 @@ def get_low_power_mode(self, port_num):
             return False
 
         try:
-            reg_file = open("/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r")
+            reg_file = open(
+                "/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r")
```

Comment:
Can you remove above link break?

Reply:
Fixed in 5db212356dfce548f532d8398de43fed8dccf973